src/test/java/uk/gov/ons/ctp/response/lib/common/UnirestInitialiser.java

```diff
@@ -2,28 +2,36 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.mashape.unirest.http.Unirest;
 import java.io.IOException;
+import kong.unirest.core.Unirest;
 
 public class UnirestInitialiser {
-  public static void initialise(final ObjectMapper mapper) {
-    Unirest.setObjectMapper(
-        new com.mashape.unirest.http.ObjectMapper() {
-          public <T> T readValue(final String value, final Class<T> valueType) {
-            try {
-              return mapper.readValue(value, valueType);
-            } catch (IOException e) {
-              throw new RuntimeException(e);
-            }
-          }
+  public static void initialise() {
 
-          public String writeValue(final Object value) {
-            try {
-              return mapper.writeValueAsString(value);
-            } catch (JsonProcessingException e) {
-              throw new RuntimeException(e);
-            }
-          }
-        });
+    ObjectMapper objectMapper = new ObjectMapper();
+
+    Unirest.config()
+        .connectTimeout(1000)
+        .setDefaultHeader("Accept", "application/json")
+        .followRedirects(false)
+        .enableCookieManagement(false)
+        .setObjectMapper(
+            new kong.unirest.core.ObjectMapper() {
+              public <T> T readValue(final String value, final Class<T> valueType) {
+                try {
+                  return objectMapper.readValue(value, valueType);
+                } catch (IOException e) {
+                  throw new RuntimeException(e);
+                }
+              }
+
+              public String writeValue(final Object value) {
+                try {
+                  return objectMapper.writeValueAsString(value);
+                } catch (JsonProcessingException e) {
+                  throw new RuntimeException(e);
+                }
+              }
+            });
```

Comment:
Still using the jackson object mapper, the the new unirest client has some other config we need to add.